BusBoard.ConsoleApp/Apis/Tfl/Responses/StopPoint.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace BusBoard.ConsoleApp
+{
+    class StopPoint
+    {
+        public double lat { get; set; }
+        public double lon { get; set; }
+        public double distance { get; set; }
+        public string id { get; set; }
+        public string commonName { get; set; }
+
+        public double LatLonDistance(double lat1, double lon1)
```

Comment:
It's best to avoid appending numbers to variable names as that can easily get confusing. Maybe `latToCompare` and `lonToCompare`? 🙂

Reply:
Yep, changed in 6d231f8 🙂